styles.css

```diff
@@ -1 +1,73 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+* {
+    padding: 0;
+    margin: 0;
+}
+
+.container {
+    background-color: #c0cbc3;
+    padding: 16px;
+    color: #463e4a;
+
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    column-gap: 16px;
+    row-gap: 16px;
+    grid-template-areas:
+        "header"
+        "text"
+        "section-c"
+        "section-d"
+        "section-e"
+        "none"
+        "footer"
+}
+
+.item {
+    background-color: #96A89C;
+    border: 5px double #6d8475;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 16px;
+}
+
```

Comment:
Dit is ook super handig! Dan hoef je dit niet bij elke section dit nog erbij te schrijven (tip voor mezelf haha). Alleen de padding: 16px hoeft mss alleen bij de #text area omdat de flex-direction etc al de rest doet